src/cli/flowr.ts

```diff
@@ -100,7 +104,20 @@ if(options['no-ansi']) {
 }
 
 export const defaultConfigFile = 'flowr.json';
-setConfigFile(options['config-file'] ?? defaultConfigFile, undefined, true);
+let usedConfig = false;
+if(options['config-json']) {
+	try {
+		const config = JSON.parse(options['config-json']) as FlowrConfigOptions;
+		log.info(`Using passed config ${JSON.stringify(config)}`);
+		setConfig(config);
```

Comment:
Could you please validate the schema here as well? maybe we just unify a more general `loadConfig` which does the validation? Maybe even migrations in the future?